Charith/example-1/assets/css/data-gathering.html

```diff
@@ -11,7 +11,41 @@
           integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 </head>
 <body>
-
+<div class="bg-info">
+    <div class="bg-info">
+        <h2 class="text-uppercase text-center text-white pb-3">Simple Contact Form</h2>
+    </div>
+    <div class="card bg-light w-50 mx-auto">
+        <div class="card-body">
+            <form class="bg-light p-3">
+                <div class="form-group text-danger"><label for="first-name">FIRST NAME</label>
+                    <input type="text" class="form-control" id="first-name" placeholder="FIRST NAME" name="first name"
+                           required>
+                </div>
+                <div class="form-group text-danger"><label for="LAST NAME">LAST NAME</label>
+                    <input type="text" class="form-control" id="LAST NAME" placeholder="LAST NAME" name="LAST NAME"
+                           required>
+                </div>
+                <div class="form-group text-danger"><label for="EMAIL">EMAIL</label>
+                    <input type="text" class="form-control" id="EMAIL" placeholder="EMAIL" name="EMAIL" required>
```

Comment:
Here you can use the input type as email. Then it will validate if user entered a valid email or not. 

Ex:
`<input type="email">`